src/gdpr-utils.ts

```diff
@@ -237,64 +197,74 @@ function _optInOut(optValue, token, options) {
     _getStorage(options).set(
         _getStorageKey(token, options),
         optValue ? 1 : 0,
-        _.isNumber(options.cookieExpiration) ? options.cookieExpiration : null,
-        !!options.crossSubdomainCookie,
-        !!options.secureCookie,
-        !!options.crossSiteCookie,
-        options.cookieDomain
+        _isNumber(options.cookieExpiration) ? options.cookieExpiration : null,
+        options.crossSubdomainCookie,
+        options.secureCookie
     )
 
     if (options.capture && optValue) {
         // only capture event if opting in (optValue=true)
-        options.capture(options.captureEventName || '$opt_in', options.captureProperties, {
-            send_immediately: true,
+        options.capture(options.captureEventName || '$opt_in', options.captureProperties || {}, {
+            send_instantly: true,
         })
     }
 }
 
+export function userOptedOut(posthog: PostHog, silenceErrors: boolean | undefined) {
```

Comment:
This is extracted from `addOptOutCheck`, making it possible to check if we need to not capture without wrapping a function with magic.